recipes/duckdb/all/patches/1.0.0-0002-msvc-bicobj.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 50d10e8..a06f30c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -537,7 +537,7 @@ if(NOT MSVC)
+   endif()
+ else()
+   set(CMAKE_CXX_WINDOWS_FLAGS
+-      "/wd4244 /wd4267 /wd4200 /wd26451 /wd26495 /D_CRT_SECURE_NO_WARNINGS /utf-8")
```

Comment:
Isn't it enough with https://github.com/duckdb/duckdb/blob/5e3c44147b2abe0509165fbcd13b926e772679ff/src/CMakeLists.txt#L41